package/network/utils/xdp-tools/Makefile

```diff
@@ -0,0 +1,129 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=xdp-tools
+PKG_RELEASE:=$(AUTORELEASE)
+PKG_VERSION:=1.2.4
+PKG_HASH:=a667f4ebab15851a3936e1917a8a85eb2e60622b539eb97cd547277402711c94
```

Comment:
The has is wrong. It should be:
`dce4838e5fe609bb422030da7103be22aebcb62502716dffa1208c80894380e7`

Reply:
Fixed. Thank you!